### ISSUE TITLE
Milestone 3 feedback

### ISSUE THREAD

**Issue by liuaaron19** [yasminenassar/cogs121#2]
<del>Milestone 3 grade: 14/15</del>

<del>-1 point lost for submitting the milestone after the due time. I pulled the version from before the due time, but that version was missing descriptions, so that's what's been deducted for. </del>

Nevermind, was a mistake on my end.

**Comment by yasminenassar** [comment]
Hello, we did commit the descriptions before noon on May 13 (we’re pretty sure we did it like midnight or 1 am of that day). Is there any way we can view the exact time we pushed? We’re positive we didn’t touch it after noon on May 13.

**Comment by liuaaron19** [comment]
Yeah, nevermind, I re-checked the push time on my end and it looks like I made a mistake and read it wrong. Apologies, your team is fine.
